include/boost/fixed_string/fixed_string.hpp

```diff
@@ -1409,6 +1409,17 @@ class fixed_string
         size_type pos = 0,
         size_type count = npos) const;
 
+    /** Returns a view of a substring.
+
+        Returns a view of a substring `(pos, pos + count)`. If the requested view is greater than the size of the string, the returned view is `[pos, size())`.
+
+        @throw std::out_of_range if `pos > size()`
+    */
+    string_view_type
+    subview(

```

Comment:
Rather than introduce new syntax I think `string_view(fs).substr(...)` where `fs` is a fixed string is entirely reasonable.